src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,117 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_START = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_START";
+    public static final String ACTION_STOP  = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP";
+    public static final String ACTION_STOP2 = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP2";
```

Comment:
Well in this case I would like to point to an argument from you from another thead:
This will be a public interface and therefore the user isn't aware of the internal structure of the app. The only thing that the user is interested in is the action. IMHO _where_ the action is executed should not be exposed in the public. Maybe it changes later?

Look at the values that usually are used in the intent filters in the android-manifest. I followed that pattern.

Whereas the last part of the String (the command) can be discussed. Maybe "START" and "STOP" are sufficient?